.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# How to contribute
```

Comment:
Let's link from the README.
For example: 
https://github.com/orca-services/cakephp-heartbeat#contributing